Include/internal/pycore_code.h

```diff
@@ -345,8 +346,8 @@ extern void _Py_Specialize_Call(_PyStackRef callable, _Py_CODEUNIT *instr,
                                 int nargs);
 extern void _Py_Specialize_CallKw(_PyStackRef callable, _Py_CODEUNIT *instr,
                                   int nargs);
-extern void _Py_Specialize_BinaryOp(_PyStackRef lhs, _PyStackRef rhs, _Py_CODEUNIT *instr,
-                                    int oparg, _PyStackRef *locals);
+extern int _Py_Specialize_BinaryOp(_PyStackRef lhs, _PyStackRef rhs, _Py_CODEUNIT *instr,
```

Comment:
Until now, specialization can never cause an error. I think we should keep that property.

Reply:
> Until now, specialization can never cause an error. I think we should keep that property.

Since I'm no longer allocating memory in the specialisation, I can revert this here. But if we decide that we do need to allocate data then we will need to support memory errors.